ClientCore/FileHelper.cs

```diff
@@ -90,5 +90,21 @@ public static void CreateHardLinkFromSource(string source, string destination, b
                 throw new PlatformNotSupportedException();
             }
         }
+
+        public static Encoding GetEncoding(string filename)
+        {
+            Encoding encoding = Encoding.UTF8;
```

Comment:
The default encoding in IniFile is `new UTF8Encoding(false)` (without BOM) instead of Encoding.UTF8 (with BOM).

Reply:
Done by 2336d2f74966e9a646000cb504ac2cf7d66af953.